### ISSUE TITLE
[MRC Message Store] Network consumer should be able to store messages on chain

### ISSUE THREAD

**Issue by shannonwells** [LibertyDSNP/mrc#23]
## Network consumer can write messages to chain

As a consumer of the network, I want to be able to 
* Store a message on chain
* Using a previously registered schema
* And know that it was stored
* And know that I could, and how I would access it

Blocked by #28 

# Description
See ["Write"](https://github.com/Liberty30/mrc/blob/main/designdocs/MESSAGE_STORAGE.md#write) section of Design Doc

# Acceptance Criteria
- [ ] Expose an extrinsic that lets user submit the message to be stored
- [ ] The message schema_id must exist (validation against the schema is #41)
- [ ] The extrinsic call emits event with sufficient info to determine success (this is going to be implemented in on_finalize or on_initialize per schema_Id. We are not going to emit an event per message since that not scalable.)
- [ ] tests for same
- [ ] benchmarks

## Not required
- [ ] transaction fee for posting message (depends on capacity and can be parallelized in another story)
